doc/users/dflt_style_changes.rst

```diff
@@ -529,6 +529,39 @@ or by setting::
 
 in your :file:`matplotlibrc` file.
 
+``bar`` and ``barh``
+====================
+
+The default value of the ``align`` kwarg for both
+`~matplotlib.Axes.bar` and `~matplotlib.Axes.barh` is changed to
+``'center'`` from ``'edge'``.
```

Comment:
Nit: this would read more smoothly with the reversed order, "from center to edge".